### PR TITLE
Add dry-run argument

### PR DESCRIPTION
**Pull request by EmteZogaf**

### CODE REVIEW

File: clean_old_versions.py [hunk 1]
```diff
@@ -44,7 +44,11 @@ def main():
                         dest="password",
```
Comment (burnettk) [inline clean_old_versions.py]: it looks like you changed the permissions on this file from 755 to 644. was that intentional? if not, can you change it back. this did break tests.

Comment (EmteZogaf) [inline clean_old_versions.py]: This was by accident. Changing it back. Thx

**Comment by EmteZogaf** [comment]
Rebased on current `master`.

**Comment by burnettk** [comment]
thanks!

**Pull request merged**